sakai-evaluation-tool/src/webapp/component-templates/hierarchy_controls.html

```diff
@@ -93,8 +94,15 @@
                     <a rsf:id="assign-groups-link" href="" style="font-size: 0.9em;">Assign Groups</a>
                 </td>
                 <td rsf:id="users-cell">
+                    <span rsf:id="assign-user-count">67</span>
+                    &nbsp;
                     <a rsf:id="assign-users-link" href="" style="font-size: 0.9em;">Assign Users</a>
                 </td>
+                <td rsf:id="rules-cell">
+                    <span rsf:id="assign-rule-count">67</span>
+                    &nbsp;
+                    <a rsf:id="assign-rules-link" href="" style="font-size: 0.9em;">Assign Rules</a>
```

Comment:
I have the same opinion here as above. Instead of fixing this one off occurrence, we should address these in a separate JIRA project wide.